core/state_processor_rip7560.go

```diff
@@ -197,6 +203,23 @@ func ApplyRip7560ValidationPhases(chainConfig *params.ChainConfig, bc ChainConte
 		evm.Config.Tracer.OnTxStart(evm.GetVMContext(), tx, common.Address{})
 	}
 
+	/*** Nonce Manager Frame ***/
+	var nonceManagerUsedGas uint64
+	if chainConfig.IsRIP7712(header.Number) {
```

Comment:
Why nested?

Reply:
Fixed. Was nested to return error message in case RIP-7712 is not enabled in chain config. Added `else` and error.